Objective-C/Internal/CBLLogSinks+Swift.h

```diff
@@ -17,9 +17,14 @@
 //  limitations under the License.
 
 #import "CBLLogSinks.h"
+#import "CBLLogTypes.h"
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface CBLLogSinks ()
 
-+ (void) resetApiVersion;
++ (void) writeSwiftLog: (CBLLogDomain)domain level: (CBLLogLevel)level message: (NSString*)message;
```

Comment:
This should be at the CBLLog not CBLLogSinks. CBLLogSinks is only for setting up the destinations where the log messages go, not for logging.